Momento/Incubating/Internal/ScsDataClient.cs

```diff
@@ -482,4 +481,101 @@ public async Task<CacheDictionaryRemoveFieldsResponse> DictionaryRemoveFieldsAsy
         }
         return new CacheDictionaryRemoveFieldsResponse();
     }
+
+    public async Task<CacheSetAddResponse> SetAddAsync(string cacheName, string setName, byte[] element, bool refreshTtl, uint? ttlSeconds = null)
+    {
+        return await SendSetAddAsync(cacheName, setName, element.ToByteString(), refreshTtl, ttlSeconds);
+    }
+
+    public async Task<CacheSetAddResponse> SetAddAsync(string cacheName, string setName, string element, bool refreshTtl, uint? ttlSeconds = null)
+    {
+        return await SendSetAddAsync(cacheName, setName, element.ToByteString(), refreshTtl, ttlSeconds);
+    }
+
+    private async Task<CacheSetAddResponse> SendSetAddAsync(string cacheName, string setName, ByteString element, bool refreshTtl, uint? ttlSeconds = null)
+    {
```

Comment:
why doesn't this just delegate to SetAddBatchAsync?

Reply:
I didn't want to wrap the single element in a list as a performance optimization. If you think that's premature I can change it.

Reply:
seems likely that `new []{ element }` won't be the slow part of this but of course you could always take the builder as a parameter to the internal method instead and completely elide the copy. I don't think it matters much. Less code is better though!

Reply:
Fixed in [9187dfb](https://github.com/momentohq/client-sdk-csharp/pull/93/commits/9187dfbfb29fcd518a1cf9de5cd1ffa6ac3c2e89)